src/malawi/get_ecmwf_seasonal_data.py

```diff
@@ -0,0 +1,52 @@
+"""
+Download raster data from GLOFAS and extracts time series of water discharge in selected locations,
+matching the FFWC stations data
+"""
+import logging
+import geopandas as gpd
+
+# TODO: remove this after making top-level
+from pathlib import Path
+import os
+import sys
+
+path_mod = f"{Path(os.path.dirname(os.path.realpath(__file__))).parents[1]}/"
+sys.path.append(path_mod)
+from src.indicators.drought import ecmwf_seasonal
+from src.indicators.flooding.glofas.area import AreaFromShape
+from src.indicators.drought.config import Config
+
+# Location of stations on the Jamuna/Brahmaputra river from http://www.ffwc.gov.bd/index.php/googlemap?id=20
+# Some lat lon indicated by FFWC are not on the river and have been manually moved to the closest pixel on the river
+# Bahadurabad_glofas corresponds to the control point identified here:
+# https://drive.google.com/file/d/1oNaavhzD2u5nZEGcEjmRn944rsQfBzfz/view
+COUNTRY_NAME = "malawi"
```

Comment:
Get these from the config file too (I know I didn't, but I should have!)

Reply:
You need to define the country name manually I would say, but the country iso3 yes, should come from the config